.cirrus.yml

```diff
@@ -124,6 +124,27 @@ task:
     - name: web_tests-7_last-linux # last Web shard must end with _last
       << : *WEB_SHARD_TEMPLATE
 
+    - name: web_integration_test_linux
```

Comment:
nit: I imagine in the future we will want to run framework-side integration tests as part of the engine build as well. To make it clear that this shard is running _engine_ tests specifically, I'd put the word "engine" somewhere in the name of the shard.